index.js

```diff
@@ -50,7 +59,21 @@ function getAllMovieTitles() {}
  *  checkIfAnyMovieHasRating(movies, "R");
  *  //> false
  */
-function checkIfAnyMovieHasRating() {}
+function checkIfAnyMovieHasRating(movies,rating = 
+  "G") {
+  if (movies.length === 0) {
+    throw "There Are No Movies Available"
+}
+return movies.some((movie) => {
+  if (
+  movie.rated === rating){
+    return true 
```

Comment:
Try returning the boolean in this condition directly... `reutrn movie.rated === rating`. If that value is true, you're returning... that value. And if it's false, you're returning... that value. If you're returning the value of your condition either way, you can just return the condition!